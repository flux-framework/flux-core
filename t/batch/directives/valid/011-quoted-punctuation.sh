#flux: --output='Test_quoted.{{id}}.out'
#flux: --output=Test_unquoted.{{id}}.out
#flux: --output=Test_unquoted.-out
#flux: --output=Test_unquoted.:out
#flux: --output=Test_unquoted.@out
#flux: --output=Test[unquoted]
