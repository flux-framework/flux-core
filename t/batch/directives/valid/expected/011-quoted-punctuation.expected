SETARGS(['--output=Test_quoted.{{id}}.out'])
SETARGS(['--output=Test_unquoted.{{id}}.out'])
SETARGS(['--output=Test_unquoted.-out'])
SETARGS(['--output=Test_unquoted.:out'])
SETARGS(['--output=Test_unquoted.@out'])
SETARGS(['--output=Test[unquoted]'])
